Outdated version of sitespeed.io is creating issues in > node 4.0

I ran into issues when trying to move to Node 4.x due to a dependency chain leading to gulp-sitespeedio.  This issue does not exist in sitespeed.io at lastest (currently 3.11.1), but gulp-sitespeedio's current version for sitespeed.io is 3.5.0.  Updating this dependency would allow for use of Node 4 and later.

For reference: 

```
/Users/{me}/dev/{project}/node_modules/gulp-sitespeedio/node_modules/sitespeed.io/node_modules/browsertime/node_modules/selenium-webdriver/node_modules/ws/node_modules/utf-8-validate
> node-gyp rebuild

  CXX(target) Release/obj.target/validation/src/validation.o
In file included from ../src/validation.cc:15:
../node_modules/nan/nan.h:261:25: error: redefinition of '_NanEnsureLocal'
NAN_INLINE v8::Local<T> _NanEnsureLocal(v8::Local<T> val) {
```

This is one of many similar errors I see when trying to use Node 4.x

--- a/package.json
+++ b/package.json
@@ -20,13 +20,12 @@
     "node": ">= 0.10.7"
   },
   "dependencies": {
-    "sitespeed.io": "3.5.0",
     "gulp-util": "^3.0.4",
     "object-assign": "~2.0.0",
+    "sitespeed.io": "^3.11.1",
     "temporary": "0.0.8"
   },
   "devDependencies": {
-    "gulp-sitespeedio": "^0.0.7",
     "gulp": "~3.8.11"
   },
   "bugs": {

